package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "marionette-client",
-  "version": "1.7.2",
```

Comment:
1.7.3 was a version used only for debugging issues with sockit-to-me and session timeouts. Had to skip to 1.7.4.